src/main/flight/pid.c

```diff
@@ -344,6 +356,9 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     itermLimit = pidProfile->itermLimit;
     throttleBoost = pidProfile->throttle_boost * 0.1f;
     itermRotation = pidProfile->iterm_rotation == 1;
+    itermRelax = pidProfile->iterm_relax == 1;
+    itermRelaxCutoffLow = pidProfile->iterm_relax_cutoff_low;
```

Comment:
`itermRelaxCutoff[2]` may make the code cleaner

Reply:
Ok, I promise to do that once I add a third filter. But with two I see it as a bit of a wash, no?